apiserver/facades/client/usermanager/usermanager.go

```diff
@@ -228,6 +280,33 @@ func (api *UserManagerAPI) enableUserImpl(args params.Entities, action string, m
 		err = method(user)
 		if err != nil {
 			result.Results[i].Error = apiservererrors.ServerError(errors.Errorf("failed to %s user: %s", action, err))
+			continue
+		}
+		// End legacy block.
+
+		// Parse User tag
+		usrTag, err := names.ParseUserTag(arg.Tag)
+		if err != nil {
+			result.Results[i].Error = apiservererrors.ServerError(err)
+			continue
+		}
+
+		// Get User
+		usr, err := api.userService.GetUserByName(ctx, usrTag.Name())
+		if err != nil {
+			result.Results[i].Error = apiservererrors.ServerError(errors.Errorf("failed to %s user: %s", action, err))
+			continue
+		}
```

Comment:
It seems we could clean this up slightly, by having a method on the `UserManagerAPI`.

```go
func (api *UserManagerAPI) getUserByTag(tag string) (coreuser.User, error) {
		tag, err := names.ParseUserTag(arg.Tag)
		if err != nil {
			return coreuser.User{}, errors.Trace(err)
		}
		return api.userService.GetUserByName(ctx, tag.Name())
}
```

Reply:
thx for the suggestion. done